src/bp/core/services/ghost/disk-driver.ts

```diff
@@ -110,6 +110,15 @@ export default class DiskStorageDriver implements StorageDriver {
     }
   }
 
+  async absoluteDirectoryListing(destination: string) {
```

Comment:
If you prefer, `walk` is also commonly used as naming for deep directory listing